src/components/Onboarding/steps/GenuineCheck/GenuineCheckUnavailable.js

```diff
@@ -29,7 +30,13 @@ export function GenuineCheckUnavailableFooter({
         {t('app:common.back')}
       </Button>
       <Box horizontal ml="auto">
-        <Button padded disabled={false} onClick={() => nextStep()} mx={2}>
+        <Button
+          padded
+          disabled={false}
+          event="Skip Genuine Check Onboarding"
```

Comment:
I think better to prefix Onboarding . 
actually i'm even not sure we need to say it's "onboarding". could just be Skip Genuine Check. because we know what is the current "page"